.travis.yml

```diff
@@ -9,6 +9,7 @@ install:
   - pip install -r requirements.txt
 script:
   - make pep8
+  - make flake8 FLAKE8_OPTS=--exit-zero
```

Comment:
Think we want to remove FLAKE8_OPTS=--exit-zero so we actually do fail travis builds on flake8 failures.